python/py-ipaddr/Portfile

```diff
@@ -3,24 +3,36 @@
 PortSystem          1.0
 PortGroup           python 1.0
 
-name                py-ipaddr
-version             2.1.9
-categories-append   www
+set real_name       ipaddr
+name                py-${real_name}
+version             2.2.0
+revision            0
+worksrcdir          ${real_name}-${version}
+distfiles           ${real_name}-${version}${extract.suffix}
```

Comment:
Instead of setting `worksrcdir` and `distfiles`, just set the `distname`.

`distname ${python.rootname}-${version}`